### PR TITLE
Feature/serpapi loader integration

### PR DESCRIPTION
**Pull request by fantastic-genius**
**Description:**
- implemented serpapi loader integration
- added its corresponding template

**Related:**
[https://github.com/FlowiseAI/Flowise/issues/652](https://github.com/FlowiseAI/Flowise/issues/652)

**Usage**
2023-08-19 - 17-05-33 - Flowise - Low-code LLM apps builder

**QnA**

2023-08-19 - 17-29-55 - Flowise - Low-code LLM apps builder
2023-08-19 - 17-30-32 - Flowise - Low-code LLM apps builder

### CODE REVIEW

**Comment by HenryHengZJ** [comment]
@fantastic-genius if you can run `yarn lint-fix`

**Comment by fantastic-genius** [comment]
> @fantastic-genius if you can run `yarn lint-fix`

Alright.  Thanks @HenryHengZJ

**Comment by fantastic-genius** [comment]
> @fantastic-genius if you can run `yarn lint-fix`

The linting is now fixed @HenryHengZJ

File: packages/server/marketplaces/chatflows/Web Search QnA.json [hunk 1]
```diff
@@ -0,0 +1,570 @@
+{
```
Comment (HenryHengZJ) [inline packages/server/marketplaces/chatflows/Web Search QnA.json:1]: thanks for the template! I think this is very similar to other existing template in marketplace like WebPageQnA and ConversationalRetrievalQAChain and etc, so maybe we can skip having a template for this

Comment (fantastic-genius) [reply packages/server/marketplaces/chatflows/Web Search QnA.json:1]: alright

**Review by HenryHengZJ — APPROVED** [review]
thank you @fantastic-genius !

**Pull request merged**